Tutorials/CAmkES/camkes-vm-linux.md

```diff
@@ -3,9 +3,12 @@ toc: true
 title: Camkes VM Linux
 tutorial: camkes-vm-linux
 tutorial-order: vm-1
+layout: tutorial
 description: using Linux as a guest in the Camkes VM.
 SPDX-License-Identifier: CC-BY-SA-4.0
 SPDX-FileCopyrightText: 2020 seL4 Project a Series of LF Projects, LLC.
 ---
 {% include tutorial.md %}
+<script src="{{ base.url | prepend: site.url }}/assets/js/toggle-markdown.js"></script>
```

Comment:
If possible we should move `script` tags and similar into the main layout template (I think `_layouts/tutorial.html` in this case)